src/constants/networks.ts

```diff
@@ -201,5 +201,32 @@ export const DEFAULT_REWARDS: { [key: string]: string[] } = {
   [ChainId.MAINNET]: ['0x9F52c8ecbEe10e00D9faaAc5Ee9Ba0fF6550F511'],
 }
 
+export const SUPPORTED_NETWORKS_FOR_MY_EARNINGS = [
+  ChainId.MAINNET,
+  ChainId.ARBITRUM,
+  ChainId.OPTIMISM,
+  // ChainId.MATIC,
```

Comment:
any update for production env

Reply:
this is final update for production. BE is only support 3 networks for now